doc/examples/selenium/package.json

```diff
@@ -1,20 +0,0 @@
-{
-  "name": "axe-webdriverjs-example",
```

Comment:
No more Webdriver example? This still seems useful, people might not know about axe-webdriverjs.

Reply:
People not being able to find their way to axe-webdriverjs is a different problem. We shouldn't teach people how to build their own, which is what this example was doing. Just use what's already out there.